test/format/format.h

```diff
@@ -60,6 +60,24 @@
 
 #define MAX_MODIFY_ENTRIES 5 /* maximum change vectors */
 
+/* 
+ * Abstract lock that lets us use either pthread reader-writer locks or WiredTiger's own
+ * (likely faster) implementation.
+ */
+typedef struct {
+    union {
+	WT_RWLOCK	 wt;
+	pthread_rwlock_t pthread;
+    } l;
+#define LOCK_NONE	0
+#define LOCK_WT    	1
+#define LOCK_PTHREAD	2
```

Comment:
Maybe an enum? And, this is zero'd memory, so if `LOCK_NONE == 0`, I don't think you need the `LOCK_CLEAR` macro any more.

Def not a unint32_t, there's no reason to use a fixed size type here?

Reply:
Done. Switched to enum and eliminated `LOCK_CLEAR()` macro.